nemo_aligner/utils/trt_llm.py

```diff
@@ -48,10 +46,9 @@ def __init__(
         repetition_penalty=1.0,
         max_generation_length=1024,
         max_input_len=1024,
-        max_input_tokens=4096,
         generation_batch_size=4,
         use_greedy=False,
-        trt_model_type="GPTForCausalLM",
+        trt_model_type="llama",
```

Comment:
is it safe to change the trt-model-type like this?

Reply:
the GPTForCausalLM was an artifact of the old naming, now we have these [gpt/llama aliases](https://github.com/NVIDIA/NeMo/blob/9d857baf82ce163801459fa7cec71c1e38016b02/nemo/export/trt_llm/converter/utils.py#L30) for those trtllm classes now. actually the [constructor caller](https://github.com/NVIDIA/NeMo-Aligner/blob/main/nemo_aligner/models/nlp/gpt/megatron_gpt_ppo_actor.py#L91) already defaulted properly so this is just to bring this up to date